Basis/Operators.swift

```diff
@@ -44,9 +44,21 @@ infix operator <| {
 ///		
 /// can also be written as:
 ///
-///		nubBy(==)(x) |> zip(y)
-///					 |> map(==)
-///					 |> and
+///		nubBy(==)(x) |>> zip(y)
+///					 |>> map(==)
+///					 |>> and
+infix operator |>> {
+	precedence 100
+	associativity left
+}
+
+/// Cons | Constructs a list by appending a given element to the front of a list.
+infix operator <| {
+	precedence 100
+	associativity right
+}
+
+/// Snoc | Constructs a list by appending a given element to the end of a list.
```

Comment:
If this the proper associativity?  I've only ever seen snoclists in Idris and they seemed to work well infixl.